__tests__/vue/click.js

```diff
@@ -0,0 +1,342 @@
+import { render, cleanup } from "@testing-library/vue";
+import "jest-dom/extend-expect";
+import userEvent from "../../src";
+
+afterEach(cleanup);
+
+describe("userEvent.click", () => {
+  it.each(["input", "textarea"])(
+    "should fire the correct events for <%s>",
+    type => {
+      const events = [];
+      const eventsHandler = jest.fn(evt => events.push(evt.type));
+      const { getByTestId } = render({
+        render: function(h) {
+          return h(type, {
+            attrs: {
+              "data-testid": "element"
+            },
+            on: {
+              mouseOver: eventsHandler,
```

Comment:
```suggestion
              mouseover: eventsHandler,
```

@afontcu If you lowercase the handler properties, then they fire correctly (applies to all the events in this file)

Reply:
@afontcu I created a PR to your branch that fixes this: https://github.com/afontcu/user-event/pull/1

Reply:
Oh wow, I didn't even think about that. Really nice catch! 🙌